app/admin/webapp/extension/UploadDialog.fragment.xml

```diff
@@ -7,7 +7,7 @@
                     id="uploader"
                     fileType="csv"
                     multiple="false"
-                    uploadUrl="/api/admin/Csv/data"
+                    uploadUrl="/api/admin/CsvStream/data"
```

Comment:
From a URL perspective I think having something like `/api/admin/upload/csv` would be nice, but I guess any naming here would be fine for me.

Reply:
Upload sounds good, too, as we are not persisting the stream anyways. So there'd be no "download". But entities in the model have to start with a capital, right? Is there a way to override the entity name for the path?

Reply:
> But entities in the model have to start with a capital, right?

I don't think anybody enforces this, but I am not sure.

> Is there a way to override the entity name for the path only?

No currently not.

Reply:
> I don't think anybody enforces this, but I am not sure.

Actually, it works. The generated class will be annotated like this:
`@CdsName("AdminService.upload")`

But do we want to promote this?

Reply:
> But do we want to promote this?

good question, not sure about that :)

Reply:
Playing it safe then :)